src/micro_http/src/server.rs

```diff
@@ -375,18 +377,21 @@ impl HttpServer {
     /// The file descriptor of the `epoll` structure can enable the server to become
     /// a non-blocking structure in an application.
     ///
-    /// Returns the file descriptor of the server's internal `epoll` structure.
+    /// Returns the instance of the server's internal `epoll` structure.
```

Comment:
nit: This should be "Returns a **reference** to ..."

Reply:
:+1: